docs/guide/core-concepts.md

```diff
@@ -425,3 +627,122 @@ describe some tiled map area in Geotrellis.
 **How are Layout Definitions used throughout Geotrellis?**
 
 They are used heavily when reading, writing, and reprojecting Rasters.
+
+
+Map Algebra
+===========
+
+Map Algebra is a name given by Dr. Dana Tomlin in the 1980's to a way of manipulating
+and transforming raster data. There is a lot of literature out there, not least
+[the book by the guy who "wrote the book" on map algebra](http://esripress.esri.com/display/index.cfm?fuseaction=display&websiteID=228&moduleID=0),
+so we will only give a brief introduction here.
+GeoTrellis follows Dana's vision of map algebra operations,
+although there are many operations that fall outside of the realm of Map Algebra that it also supports.
+
+**Categories of Map Algebra operations**
+
+Map Algebra operations fall into 3 general categories:
+
+**Local**
+
+
+
+![Local Operations](./images/local-animations-optimized.gif localops)
```

Comment:
they were working for me. and I have uploaded them. might be a weird pr issue. can confirm after its in. and adjust.